assets/css/card.css

```diff
@@ -329,3 +311,183 @@
   font-size: 1.5rem;
   background-color: var(--white-shade);
 }
+
+@media only screen and (max-width: 480px) {
+  .Card-container {
+    width: 100%;
+  }
+
+  .Card-container-shadow {
+    width: 100%;
+  }
+
+  .Card-one {
+    width: 100%;
+  }
+
+  .Card-two {
+    width: 100%;
+  }
+
```

Comment:
Suggestion: You could combine all three classes in one if they posses same properties.

.classone, 
.classtwo,
.classthree {
    width:100%;
}